src/codegen/tests/debug_tests/snapshots/rusty__codegen__tests__debug_tests__expression_debugging__for_conditions_location_marked.snap

```diff
@@ -17,11 +17,9 @@ condition_check:                                  ; preds = %increment, %entry
   %load_myFunc = load i32, i32* %myFunc, align 4, !dbg !11
   %load_myFunc1 = load i32, i32* %myFunc, align 4, !dbg !11
   %tmpVar = icmp sle i32 %load_myFunc1, 20, !dbg !11
-  %0 = zext i1 %tmpVar to i8, !dbg !11
-  %1 = icmp ne i8 %0, 0, !dbg !11
```

Comment:
This feels strange to me, we do this for bools, so maybe we missed something here?

Reply:
I'm pretty sure this is because for booleans we no longer hit this [section]((https://github.com/PLC-lang/rusty/blob/9037523a2ef45ba82eac0f6b7ec8f6c4903e7234/src/codegen/llvm_typesystem.rs#L156C1-L170C14))

Reply:
Reverting back to the initial naive fix implementation because of https://github.com/PLC-lang/rusty/commit/5fb7b1c400047244262f33f478e3c3c15df27093